packages/p2p/src/components/composite-calendar/composite-calendar.jsx

```diff
@@ -0,0 +1,201 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Loadable from 'react-loadable';
+import { DesktopWrapper, InputField, MobileWrapper, useOnClickOutside } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { daysFromTodayTo, epochToMoment, toMoment } from '@deriv/shared';
+import CompositeCalendarMobile from './composite-calendar-mobile.jsx';
+import SideList from './calendar-side-list.jsx';
+import CalendarIcon from './calendar-icon.jsx';
+import { observer } from 'mobx-react-lite';
+import { useStore } from '@deriv/stores';
+
+const TwoMonthPicker = Loadable({
+    loader: () => import(/* webpackChunkName: "two-month-picker" */ './two-month-picker.jsx'),
+    loading: () => null,
+    render(loaded, props) {
+        const Component = loaded.default;
+        return <Component {...props} />;
+    },
+});
+
+const CompositeCalendar = props => {
+    const { onChange, to, from } = props;
+    const {
+        ui: { current_focus, setCurrentFocus },
+    } = useStore();
+
+    const [show_to, setShowTo] = React.useState(false);
+    const [show_from, setShowFrom] = React.useState(false);
+    const list = [
```

Comment:
could we give the list a different name such as the one below? just so others can understand what this list is
```suggestion
    const time_duration_list = [
```